pylint/message/message_definition.py

```diff
@@ -25,23 +26,21 @@ def __init__(
         msg: str,
         description: str,
         symbol: str,
-        scope: str,
-        minversion: tuple[int, int] | None = None,
-        maxversion: tuple[int, int] | None = None,
-        old_names: list[tuple[str, str]] | None = None,
+        extra_options: ExtraMessageOptions,
```

Comment:
Is this public/private? I'm not sure we can change this signature without a changelog warning.